client/overview/index.js

```diff
@@ -101,6 +102,8 @@ const OverviewPage = () => {
 				</Notice>
 			) }
 
+			<JetpackIdcNotice />
```

Comment:
Not sure what the standard is but does it make sense to move the conditional logic here and then have the component always render the notice with the callee responsible for conditional rendering?
Asking since the conditions we use are part of the global `wcpaySettings` object and we aren't passing down any props to the component itself anyways.

Reply:
This is a good point! However, I am following up the same pattern that `TestModeNotice` component (right below this line) is using.

https://github.com/Automattic/woocommerce-payments/blob/7b42c8b8dfdeb778a8a35eb399acd65dd9c50afe/client/components/test-mode-notice/index.js#L105-L106

I think it has an advantage that we can pick up this component to other pages like Transactions, Disputes, etc. without making so many change and condition.